payroll/views/base_time_parameter_views.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="base_time_parameter_action" model="ir.actions.act_window">
+        <field name="name">Time Parameters</field>
```

Comment:
Last suggestion. Can we mantain the name of the view as "Salary rule time parameter" or something similar? 

It's important to me that my current users of the module adapt to this change with the minor hassle possible. Apart of that, I like the change, will test it when the other module PR is merged and we can merge here.

Reply:
Done

Reply:
@norlinhenrik Thanks Henrik, will test everything today and let you know.